src/server/dev.ts

```diff
@@ -12,7 +12,9 @@ import webpackConfig from '../../webpack.config'
 import { RobotSimulator } from '../simulators/robot_simulator'
 import { SimulatorStatus } from '../simulators/robot_simulator'
 import { SensorDataSimulator } from '../simulators/sensor_data_simulator'
+import { ChartSimulator } from '../simulators/chart_simulator'
 import { NUSightServer } from './app/server'
+import { Chart } from '../client/components/chart/view'
```

Comment:
hmm, yep, must of been IntelliJ auto importing when I wrote Chart instead of ChartSimulator or something.